containers/Investment/RevolvingPool/View/CollectCard.tsx

```diff
@@ -25,6 +27,22 @@ const CollectCard: React.FC<Props> = (props: Props) => {
 
   const [status, , setTxId] = useTransactionState()
 
+  // If it's a redeem order, then convert amount back into DROP/TIN
+  const settledAmount =
+    type === 'Invest'
+      ? props.disbursements.payoutTokenAmount
+      : new BN(props.disbursements.payoutCurrencyAmount)
+          .div(new BN(props.tokenPrice).div(new BN(10).pow(new BN(9))))
+          .mul(new BN(10).pow(new BN(18)))
+          .toString()
```

Comment:
Why do we need to do a conversion here? If the user redeems TIN/DROP, we should rather show the amount in DAI, right? Or is props.tokenPrice the actual price at the time of execution? If it is not the price from the execution, the numbers here will not be accurate, right?

Reply:
This is what I was discussing with Dennis during our call: the settled amount should be the token amount. The DAI amount for redeem orders is shown below in the "Transaction value" row.

Reply:
And yes, the price can change and that might be different, but Dennis said that was okay.

Reply:
Okay, not sure this makes sense, but let's see whether it causes any issues. CC @denniswell @lucasvo in case you are stumbling across that.